src/scene/scene.cpp

```diff
@@ -0,0 +1,25 @@
+#include "scene.h"
+
+void Scene::addTriangle(glm::vec3 p0, glm::vec3 p1, glm::vec3 p2, glm::vec3 color) 
+{
+  Triangle tri(p0, p1, p2, mTriangleList.size());
+  tri.setColor(color);
+
+  mTriangleList.push_back(tri);
+}
+
+Hit Scene::getRayIntersection(const Ray &ray) 
+{
```

Comment:
run clang format these should be `BreakBeforeBraces: Attach`  so on the function method header.